lib/lambda_wrap/lambda_manager.rb

```diff
@@ -91,7 +91,7 @@ def deploy_lambda(bucket, key, version_id, function_name, handler, lambda_role,
 				func_version = func_config.version
 				raise 'Error while publishing existing lambda function ' + function_name if !func_config.version
 			rescue Aws::Lambda::Errors::ResourceNotFoundException
-				func_config = @client.create_function({function_name: function_name, runtime: 'nodejs', role: lambda_role, handler: handler, code: { s3_bucket: bucket, s3_key: key }, timeout: 5, memory_size: 128, publish: true, description: lambda_description}).data
+				func_config = @client.create_function({function_name: function_name, runtime: 'nodejs43', role: lambda_role, handler: handler, code: { s3_bucket: bucket, s3_key: key }, timeout: 5, memory_size: 128, publish: true, description: lambda_description}).data

```

Comment:
According to the [documentation](http://docs.aws.amazon.com/sdkforruby/api/Aws/Lambda/Client.html#create_function-instance_method), the value should be `nodejs4.3` (note the `.` in the value). Otherwise, it's a good addition and that should just be the default for LambdaWrap.